src/SOS/SOS.Package/SOS.Symbol.Package.csproj

```diff
@@ -15,7 +15,7 @@
 
   <ItemGroup>
     <None Include="$(ArtifactsBinDir)\SOS.Extensions\$(Configuration)\netstandard2.0\publish\*.pdb" Pack="true" Visible="false">
-      <PackagePath>$(SOSPackagePathPrefix)</PackagePath>
+      <PackagePath>$(SOSPackagePathPrefix)/lib</PackagePath>
```

Comment:
Is there any reason we still need a split `src/SOS/SOS.Package/SOS.Symbol.Package.csproj`? Is this essentially the same as SOS.Package with PDB? Or is it that without dlls?

Reply:
It is just the PDB's without the DLLs.

Reply:
We can revisit and change the way these two SOS packages are built in the future.